plugins/modules/win_domain_group_membership.ps1

```diff
@@ -50,7 +50,11 @@ if ($diff_mode) {
     $result.diff = @{}
 }
 
-$members_before = Get-AdGroupMember -Identity $ADGroup @extra_args
+$group = Get-ADGroup $ADGroup @extra_args
```

Comment:
No need to run this again, you can just use `$ADGroup` which was retrieved above.

Reply:
you are right. I've fixed it